examples/gym/gym_state.py

```diff
@@ -28,7 +29,7 @@ def __repr__(self) -> str:
 
     def equals(self, other: any) -> bool:
         return type(self) == type(other) and \
-               self._env.get_obs() == other._env.get_obs()   # legit?
+               np.all(np.isclose(self._env.get_obs(), other._env.get_obs()))   # legit?
```

Comment:
verify with the documentation, but i think `np.isclose(x, y)` works for arrays (no need for np.all, too)

Reply:
Verified, but changed it to `np.allclose` that basically combines both